### ISSUE TITLE
Xiaomi Air Purifier 4 Lite (zhimi.airp.rma2) support

### ISSUE THREAD

**Issue by gvillo** [dhewg/esphome-miot#14]
Hi! I was wondering if it supports `zhimi.airp.rma2` (https://home.miot-spec.com/s/zhimi.airp.rma2), I don't know exactly what are the differences with the supported `rmb1`

**Comment by kyutov** [comment]
There are small difference between the two models. You can compare the JSONs with the specs of the two models and see the differences.

[zhimi-rmb1 specs](https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:air-purifier:0000A007:zhimi-rmb1:2)
[zhimi-rma2 specs](https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:air-purifier:0000A007:zhimi-rma2:1)

**Comment by helgek** [comment]
@kyutov Do you maybe know where I can see which revision my device is after flashing ESPHome? I flashed the device straight without checking beforehand. Just asking in case it is the zhimi.airp.rma2 revision I could then confirm that it works with the zhimi.airp.rmb1.yaml config.

**Comment by dhewg** [comment]
Yeah, there're some property differences, so that one needs its own config.
If you have a backup of the vendor fw, you should find the model as string in there

**Comment by helgek** [comment]
@dhewg Thank you, yes I was able to find the ot_config.psm_modelzhimi.airp.rmb1 string in the backup of the original firmware. So I was lucky to have the right device for the already existing ESPHome configuration.

**Comment by dhewg** [comment]
Closing, this never was about `rma2`, but the already supported `rmb1`.
